src/utils/getTotalMissedUpdate.ts

```diff
@@ -0,0 +1,17 @@
+export function getTotalMissedUpdate(date: Array<number>): number {
```

Comment:
here totalMissedUpdate = (currentStandupDate - lastStanupDate) / (24 * 60 * 60 * 1000);. The problem is that the function is dividing the difference between the current date and the last standup date by the number of milliseconds in a day. However, the date array is not guaranteed to contain all of the standup dates. If the array does not contain all of the standup dates, then the function will divide by a number that is smaller than the number of milliseconds in a day. This will result in an incorrect value for totalMissedUpdate

Reply:
use below code

export function getTotalMissedUpdate(date: Array<number>): number {
    const currentStandupDate = new Date().setUTCHours(0, 0, 0, 0);
    let lastStanupDate = 0;
    let totalMissedUpdate = 0;
    if (date.length === 0) {
        return totalMissedUpdate;
    }
    for (let i = 0; i < date.length; i++) {
        if (date[i] > lastStanupDate) {
            lastStanupDate = date[i];
        }
    }
    totalMissedUpdate = Math.floor((currentStandupDate - lastStanupDate) / (24 * 60 * 60 * 1000));
    return totalMissedUpdate;
}

Reply:
fixed